grpc/server/src/main/java/io/helidon/grpc/server/GrpcServer.java

```diff
@@ -45,6 +46,13 @@
      */
     GrpcServerConfiguration configuration();
 
+    /**
+     * Gets a {@link GrpcServer} context.
+     *
+     * @return a server context
+     */
+    ContextualRegistry context();
```

Comment:
This should not use `ContextualRegistry` but `Context` - `ContextualRegistry` is used for backward compatibility in WebServer